src/fidesops/ops/models/privacy_request.py

```diff
@@ -212,6 +220,7 @@ class PrivacyRequest(Base):  # pylint: disable=R0904
     paused_at = Column(DateTime(timezone=True), nullable=True)
     identity_verified_at = Column(DateTime(timezone=True), nullable=True)
     due_date = Column(DateTime(timezone=True), nullable=True)
+    days_left = property(get_days_left)
```

Comment:
I am guessing it has something to do with sqlalchemy being unhappy, but is there a reason this was defined as a property this way instead of:

```py
@property
def days_left(self)...
```

Reply:
There isn't a specific reason. I'll try this out and see if it works. The `property` function/decorator is newer to me so I didn't realize I could do it that way.